hasher-matcher-actioner/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -165,6 +167,8 @@ def lookup_signal_with_distance(
         {
             "content_id": m.metadata,
             "distance": m.similarity_info.pretty_str(),
+            "signal_type": signal_type_name,
+            "signal_value": signal,
```

Comment:
blocking: They want to the value of the matched signal (which would need another fetch on the db for content_id), not the value they passed in. They already know the value they passed in :P 

Propose we leave this off this PR and do it after we figure out the content GET first.